### DISCUSSION TITLE
GitHub application: support gist

### DISCUSSION THREAD

**Discussion by mirsella** [community/community#6466] [Mobile] [answered]
would useful to be able to manage gist, like create delete edit and such, thanks

**Comment by amrehman** [comment, marked as answer]
Hi @mirsella ,Thanks a lot for your feedback.

I've added your note to our internal issue where we are tracking adding gists. If you have any other feedback or feature requests, please keep them coming!

**Reply by Sussybabe67** [reply]
Thanks for the app really appreciate

**Comment by AlexHedley** [comment]
#5332
#5614
#5921

**Reply by Sussybabe67** [reply]
Good one

**Comment by wai98765432** [comment]
i'm
